fdbclient/ServerKnobs.cpp

```diff
@@ -869,6 +869,7 @@ void ServerKnobs::initialize(Randomize randomize, ClientKnobs* clientKnobs, IsSi
 	init( PERSIST_FINISH_AUDIT_COUNT,                             10 ); if ( isSimulated ) PERSIST_FINISH_AUDIT_COUNT = 1;
 	init( AUDIT_RETRY_COUNT_MAX,                                 100 ); if ( isSimulated ) AUDIT_RETRY_COUNT_MAX = 10;
 	init( SS_AUDIT_AUTO_PROCEED_COUNT_MAX,                         5 );
+	init( CONCURRENT_AUDIT_TASK_COUNT_MAX,                        50 ); if ( isSimulated ) CONCURRENT_AUDIT_TASK_COUNT_MAX = 2;
```

Comment:
is it possible to make `CONCURRENT_AUDIT_TASK_COUNT_MAX` a Radom number in the range of [1, CONCURRENT_AUDIT_TASK_COUNT_MAX] in simulation?

Reply:
Agree. This could be more comprehensive.